mm2src/coins/z_coin/z_rpc.rs

```diff
@@ -167,6 +189,7 @@ impl ZRpcOps for LightRpcClient {
     }
 }
 
+#[cfg(not(target_arch = "wasm32"))]
 #[async_trait]
 impl ZRpcOps for NativeClient {
```

Comment:
Great work!
Note/Question:  did you forget to add `#[cfg(target_arch = "wasm32")]` for `impl ZRpcOps for NativeClient` and `impl ZRpcOps for LightRpcClient` or you have some plan for the future so you left just `#[cfg(not(target_arch = "wasm32"))]`target for ZRpcOps?

Reply:
Yeah, you're right. And we need to implement the storages..

Reply:
Sorry, could you clarify :D you are going yo add todos for ZRpcOps in this pr or later?
Yeah, I see that some trait methods interact with db, so it will be easier to understand what needs to be done after adding wasm storage support.
what I mean is yeah, db should be added first. You want to do it in next pr?